corva/models/stream.py

```diff
@@ -89,20 +131,32 @@ def from_raw_event(event: Union[str, List], **kwargs) -> List[StreamEvent]:
         if isinstance(event, str):
             parse = pydantic.parse_raw_as
 
-        events = parse(List[StreamEvent], event)  # type: List[StreamEvent]
+        event_dicts = parse(List[dict], event)  # type: List[dict]
```

Comment:
I thought the previous way of processing these was nicer. Why did we change away from models?

Reply:
The model is created below, the dict is used first to inject an app_key into the event